src/components/pages/Site/Site.js

```diff
@@ -352,11 +352,14 @@ const Site = ({ isNewSite }) => {
             })
           }
           if (!isAppOnline) {
-            showSyncToastError({
```

Comment:
again, if we're offline, we wont be syncing, so just show a normal toast, not the special sync toast